migration/migrator/migrations/course/20240205154116_add_poll_timer.py

```diff
@@ -0,0 +1,36 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("ALTER TABLE polls ADD COLUMN IF NOT EXISTS duration VARCHAR(255) DEFAULT 'P0D'")
+    database.execute("ALTER TABLE polls ADD COLUMN IF NOT EXISTS end_date timestamptz NOT NULL DEFAULT '9999-01-01'")
```

Comment:
Could you combine into one execute statement?

Reply:
I will leave it at is, because I also added another thing and it might be complicated to see it all in one execute.

Reply:
Try updating the migration to a single transaction for up and down methods.

Reply:
I don't understand what you mean.
Do you mean making it into one execute statement for up and down, I talked to William previously and I think we decided it would make the one line very messy. The multiple lines are for readability.